src/TheThingsNetwork.cpp

```diff
@@ -731,15 +952,15 @@ void TheThingsNetwork::configureUS915(uint8_t fsb)
   {
     if (ch == ch500 || (ch <= chHigh && ch >= chLow))
```

Comment:
Same here I think; in OTAA, the end device should try sending join-requests in all FSBs, and the actual channels are conveyed as part of the join-accept and subsequent MAC commands.

Reply:
Agree, but we need to keep defaults for up and down, even in OTAA

Reply:
How do you mean?

Reply:
@johanstokking for example in the IN865_867 band, the default channels 1-3 are disabled and others overwritten. I don't know how OTAA join works there, but it might need a different default configuration to work. Just guessing.